_data/cryptocurrencies.yml

```diff
@@ -221,3 +221,11 @@ websites:
       tfa: Yes
       software: Yes
       doc: https://twitter.com/jstcoin/status/445900548743897088
+    - name: BITLISH
+      img: bitlish.png
+      url: https://bitlish.com
+      img: btc-e.png
```

Comment:
This img tag points to `btc-e.png` and not `bitlish.png`.
bitlish.png is also larger than we allow. Here's a compressed version of the image:
![bitlish compressed](https://cloud.githubusercontent.com/assets/3535780/13024295/8c0ed8de-d1ea-11e5-9ccb-fb5f2def1ba5.png)